[2024-08-03T11:16:48.546+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-03T11:16:48.560+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_with_s3_test.copy_table_dim_products manual__2024-08-03T04:16:41.218625+00:00 [queued]>
[2024-08-03T11:16:48.566+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_with_s3_test.copy_table_dim_products manual__2024-08-03T04:16:41.218625+00:00 [queued]>
[2024-08-03T11:16:48.566+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-03T11:16:48.592+0700] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): copy_table_dim_products> on 2024-08-03 04:16:41.218625+00:00
[2024-08-03T11:16:48.595+0700] {standard_task_runner.py:64} INFO - Started process 28128 to run task
[2024-08-03T11:16:48.597+0700] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_with_s3_test', 'copy_table_dim_products', 'manual__2024-08-03T04:16:41.218625+00:00', '--job-id', '200', '--raw', '--subdir', 'DAGS_FOLDER/postgres_with_s3_test.py', '--cfg-path', '/tmp/tmph84_6d6r']
[2024-08-03T11:16:48.598+0700] {standard_task_runner.py:91} INFO - Job 200: Subtask copy_table_dim_products
[2024-08-03T11:16:48.636+0700] {task_command.py:426} INFO - Running <TaskInstance: postgres_with_s3_test.copy_table_dim_products manual__2024-08-03T04:16:41.218625+00:00 [running]> on host DESKTOP-RHVHV7A.
[2024-08-03T11:16:48.721+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='postgres_with_s3_test' AIRFLOW_CTX_TASK_ID='copy_table_dim_products' AIRFLOW_CTX_EXECUTION_DATE='2024-08-03T04:16:41.218625+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-03T04:16:41.218625+00:00'
[2024-08-03T11:16:48.721+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-03T11:16:48.730+0700] {sql.py:282} INFO - Executing: 
        COPY dim_products TO 'F:/AllCode/Data Engineer/dim_products.csv' CSV HEADER;
        
[2024-08-03T11:16:48.734+0700] {base.py:84} INFO - Using connection ID 'postgres_supermarket' for task execution.
[2024-08-03T11:16:48.750+0700] {base.py:84} INFO - Using connection ID 'postgres_supermarket' for task execution.
[2024-08-03T11:16:48.781+0700] {sql.py:495} INFO - Running statement: 
        COPY dim_products TO 'F:/AllCode/Data Engineer/dim_products.csv' CSV HEADER;
        , parameters: None
[2024-08-03T11:16:48.783+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-03T11:16:48.783+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/operators/sql.py", line 288, in execute
    output = hook.run(
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/hooks/sql.py", line 442, in run
    self._run_command(cur, sql_statement, parameters)
  File "/usr/local/lib/python3.10/dist-packages/airflow/providers/common/sql/hooks/sql.py", line 500, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.UndefinedTable: relation "dim_products" does not exist

[2024-08-03T11:16:48.786+0700] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=postgres_with_s3_test, task_id=copy_table_dim_products, run_id=manual__2024-08-03T04:16:41.218625+00:00, execution_date=20240803T041641, start_date=20240803T041648, end_date=20240803T041648
[2024-08-03T11:16:48.808+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 200 for task copy_table_dim_products (relation "dim_products" does not exist
; 28128)
[2024-08-03T11:16:48.851+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-03T11:16:48.859+0700] {logging_mixin.py:188} WARNING - /usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2024-08-03T11:16:48.868+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-03T11:16:48.890+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
